### PR TITLE
Ice: fix build on ppc64 targets

### PR DESCRIPTION
**Pull request by q66**

### CODE REVIEW

File: srcpkgs/Ice/patches/archs.patch [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+ 
+ private:
+--- cpp/src/IceUtil/Makefile
++++ cpp/src/IceUtil/Makefile
+@@ -10,7 +10,7 @@
+ top_srcdir	= ../..
+ 
+ LIBFILENAME	= $(call mklibfilename,IceUtil,$(VERSION))
+-SONAME		= $(call mksoname,IceUtil,$(SOVERSION))  
++SONAME		= $(call mksoname,IceUtil,$(SOVERSION))
```
Comment (maxice8) [inline srcpkgs/Ice/patches/archs.patch]: are these changes that just (apparently) remove extra whitespace necessary ?

Comment (q66) [reply srcpkgs/Ice/patches/archs.patch]: good catch, this one was pulled from another distro, I guess I can make it smaller

Comment (q66) [reply srcpkgs/Ice/patches/archs.patch]: fixed

**Comment by maxice8** [comment]
On Mon, Jan 07, 2019 at 10:33:03AM -0800, Daniel Kolesa wrote:
> 
> good catch, this one was pulled from another distro, I guess I can make it smaller
> 

If possible then please do most to keep them small

**Comment by q66** [comment]
I try, but I missed this one. It's now reduced to being as small as possible.

**Pull request merged**